lib/utils.js

```diff
@@ -0,0 +1,13 @@
+function countObjectsAtEndOfArray(array) {
```

Comment:
I'm not quite getting this... what about objects at the beginning or the middle of the array?